cypress/support/base.component.ts

```diff
@@ -179,4 +179,15 @@ export abstract class BaseComponent {
       .invoke('text')
       .should('to.contains', expectedTxt);
   }
+
+  isElemTextCorrect(baseSelector: string, itemSel: string, textToCheck: string, rowNum?: number) {
```

Comment:
... ( ... expectedText: string, rowNum = 0) {
cy.get(baseSelector).find(itemSel).eq(rowNum).invoke('text')
        .should('contain', expectedText);
}